test/integration/spec/connect.js

```diff
@@ -618,12 +637,18 @@ describe('connect', function() {
             });
           });
 
-          it(`should set the Local${capitalize(kind)}TrackPublication's .name to its ${names[kind] ? 'given name' : 'ID'}`, () => {
+          it(`should set the Local${capitalize(kind)}TrackPublication's .trackName to its ${names[kind] ? 'given name' : 'ID'}`, () => {
```

Comment:
Was this a typo in the test or are we changing API? Looks like a typo.

Reply:
@markandrus This was a typo.